routers/repo/http.go

```diff
@@ -528,3 +541,26 @@ func HTTPBackend(ctx *context.Context, cfg *serviceConfig) http.HandlerFunc {
 		return
 	}
 }
+
+func checkOAuthAccessToken(accessToken string) int64 {
```

Comment:
Why not return `(int64, error)`?

Reply:
This function is a copy paste from [modules/auth/auth.go](https://github.com/go-gitea/gitea/blob/master/modules/auth/auth.go#L88), I've removed this duplication and ended up just calling the original function.